[2023-03-21 01:29:05,134] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_oetker_app.run_data_metrics manual__2023-03-21T01:28:50.071119+00:00 [queued]>
[2023-03-21 01:29:05,148] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_oetker_app.run_data_metrics manual__2023-03-21T01:28:50.071119+00:00 [queued]>
[2023-03-21 01:29:05,149] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 01:29:05,150] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-21 01:29:05,150] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 01:29:05,167] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_data_metrics> on 2023-03-21 01:28:50.071119+00:00
[2023-03-21 01:29:05,173] {standard_task_runner.py:52} INFO - Started process 924 to run task
[2023-03-21 01:29:05,178] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_oetker_app', 'run_data_metrics', 'manual__2023-03-21T01:28:50.071119+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag_oetker_app.py', '--cfg-path', '/tmp/tmph593q0iu', '--error-file', '/tmp/tmpdx795qjf']
[2023-03-21 01:29:05,179] {standard_task_runner.py:80} INFO - Job 10: Subtask run_data_metrics
[2023-03-21 01:29:05,242] {task_command.py:369} INFO - Running <TaskInstance: dag_oetker_app.run_data_metrics manual__2023-03-21T01:28:50.071119+00:00 [running]> on host ec81750b2a36
[2023-03-21 01:29:05,311] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=julia
AIRFLOW_CTX_DAG_ID=dag_oetker_app
AIRFLOW_CTX_TASK_ID=run_data_metrics
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T01:28:50.071119+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T01:28:50.071119+00:00
[2023-03-21 01:29:14,782] {logging_mixin.py:115} INFO - --------------------------------------------------------------------------------------------------
[2023-03-21 01:29:14,784] {logging_mixin.py:115} INFO - Unique users: 1000
[2023-03-21 01:29:14,787] {logging_mixin.py:115} INFO - Most represented country: China with 202 users
[2023-03-21 01:29:14,788] {logging_mixin.py:115} INFO - Least represented country: Paraguay with 1 users
[2023-03-21 01:29:14,789] {logging_mixin.py:115} INFO - --------------------------------------------------------------------------------------------------
[2023-03-21 01:29:14,800] {python.py:173} INFO - Done. Returned value was: None
[2023-03-21 01:29:14,821] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_oetker_app, task_id=run_data_metrics, execution_date=20230321T012850, start_date=20230321T012905, end_date=20230321T012914
[2023-03-21 01:29:14,888] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-21 01:29:14,942] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
